Index: home-work-4/ამოცან1.py
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>''''\r\n\r\nდაწერეთ ფუნქცია, რომელიც დაადგენს გადაცემული რიცხვი მარტივია თუ არა და დააბრუნებს True-ს მარტივის\r\nშემთხვევაში და False-ს შედგენილის შემთხვევაში.\r\nფუნქციის ქვევით გამოიძახეთ ის რამდენიმე რიცხვისთვის და დაბეჭდეთ შედეგი.\r\n\r\n'''\r\n\r\ndef simple_or_complex(num):\r\n    c =0\r\n    for i in range(1, num+1):\r\n        if num % i == 0:\r\n            c+=1\r\n    if c == 2:\r\n        print(f\" რიცხვი {num} არის მარტივი \")\r\n    else:\r\n        print(f\" რიცხვი {num} არის შედგენილი \")\r\n\r\nnum =  input(\"Enter integer: \")\r\nnum = int(num)\r\nif int(num) and num > 0:\r\n    simple_or_complex(num)\r\nelse:\r\n    while(num < 0):\r\n        num = input(\"Enter integer: \")\r\n    simple_or_complex(num)\r\n
===================================================================
diff --git a/home-work-4/ამოცან1.py b/home-work-4/ამოცან1.py
--- a/home-work-4/ამოცან1.py	(revision f5ced71709ffa38d8edb3de9026006a01761e9f1)
+++ b/home-work-4/ამოცან1.py	(date 1765388526825)
@@ -12,9 +12,9 @@
         if num % i == 0:
             c+=1
     if c == 2:
-        print(f" რიცხვი {num} არის მარტივი ")
+        print(f"რიცხვი {num} არის მარტივი ")
     else:
-        print(f" რიცხვი {num} არის შედგენილი ")
+        print(f"რიცხვი {num} არის შედგენილი ")
 
 num =  input("Enter integer: ")
 num = int(num)
Index: home-work-4/GCD.py
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>def gcd(a, b):\r\n    copy_of_a = a\r\n    copy_of_b = b\r\n    while (b != 0):\r\n            tmp = a % b\r\n            a = b\r\n            b = tmp\r\n    return a\r\n\r\nif __name__ == \"__main__\":\r\n    copy_of_a = 12\r\n    copy_of_b = 8\r\n    print(f\"LCM of {copy_of_a} and {copy_of_b} is {gcd(copy_of_a, copy_of_b)}.\")
===================================================================
diff --git a/home-work-4/GCD.py b/home-work-4/GCD.py
--- a/home-work-4/GCD.py	(revision f5ced71709ffa38d8edb3de9026006a01761e9f1)
+++ b/home-work-4/GCD.py	(date 1765388490244)
@@ -10,4 +10,4 @@
 if __name__ == "__main__":
     copy_of_a = 12
     copy_of_b = 8
-    print(f"LCM of {copy_of_a} and {copy_of_b} is {gcd(copy_of_a, copy_of_b)}.")
\ No newline at end of file
+    print(f"GCD of {copy_of_a} and {copy_of_b} is {gcd(copy_of_a, copy_of_b)}.")
\ No newline at end of file
